lib/ex_force.ex

```diff
@@ -14,6 +14,19 @@ defmodule ExForce do
   end
   ```
 
+  Configure default settings in config/config.exs (optional).
+
+  ```elixir
+  # config/config.exs
+
+  config :ex_force, :api_version, "43.0"
+  config :ex_force, :adapter, {Tesla.Adapter.Hackney, [recv_timeout: 1_000]}
+  config :ex_force, :middleware, [
+    Tesla.Middleware.Telemetry,
+    {Tesla.Middleware.Timeout, timeout: :timer.seconds(1)}
+  ]
+  ```
+
```

Comment:
I'd like to avoid adding package level config like this. This can be done by having own application config and wrapper around ExForce.

Reply:
Many Elixir libraries support application-level config. This is optional and can be overridden for each client if need be.

Reply:
Updated to:

```elixir
# config/config.exs

config :ex_force, ExForce.Client.Tesla,
  api_version: "43.0",
  adapter: {Tesla.Adapter.Hackney, [recv_timeout: 1_000]},
  append_middleware: [
    Tesla.Middleware.Telemetry,
    {Tesla.Middleware.Timeout, timeout: :timer.seconds(1)}
  ]
```